src/control/setup.py

```diff
@@ -10,7 +10,7 @@
         ("share/ament_index/resource_index/packages", ["resource/" + package_name]),
         ("share/" + package_name, ["package.xml"]),
     ],
-    install_requires=["setuptools"],
+    install_requires=["setuptools", "spatialmath-python"],
```

Comment:
No need to add this here as long as it's in the root requirements.txt

Reply:
Resolved